pkg/remote/remote_test.go

```diff
@@ -211,12 +179,11 @@ func TestWithDetection_SshConfig_UserKnownHostsFile_Mismatch(t *testing.T) {
 	}
 
 	opts := []heartbeat.HandleOption{
-		remote.WithDetection(),
 		filter.WithFiltering(filter.Config{
-			Exclude:                    nil,
-			Include:                    nil,
-			IncludeOnlyWithProjectFile: false,
+			IncludeOnlyWithProjectFile: true,
 		}),
+		remote.WithDetection(),
+		remote.WithCleanup(),
```

Comment:
The `testing` package takes care of deleting temp files and directories at the end. So we can safely remove `remote.WithCleanup()`.